dm/templates/resource_policy/README.md

```diff
@@ -0,0 +1,64 @@
+# Resource Policy
+
+This template creates a resource policy.
+
+## Prerequisites
+
+- Install [gcloud](https://cloud.google.com/sdk)
+- Create a [GCP project, set up billing, enable requisite APIs](../project/README.md)
+- Enable the [Cloud Build API](https://cloud.google.com/cloud-build/docs/api/reference/rest/)

```

Comment:
Is it really the Build API we need, not the compute?

Reply:
Fixed.